### DISCUSSION TITLE
why  can't i set my username.

### DISCUSSION THREAD

**Discussion by kavindusithara** [cssbattle/forum#138] [Bugs reports]
why  can't i set my username.

**Comment by chinchang** [comment]
Have you already set up once by any chance?

**Reply by kavindusithara** [reply]
first time. but i can't set usernam
![download (10)](https://user-images.githubusercontent.com/86042234/137594007-05d28f8b-53cb-4a33-a1b2-92fcada485a5.png)
e.

**Reply by kavindusithara** [reply]
if i click set username ,but it is not set.

**Comment by DatMayo** [comment]
I have the same issue, signed up via github but button stays grayed out

**Reply by chinchang** [reply]
If its grayed out, then you already have a username set once. Do you have a username set already?

**Reply by DatMayo** [reply]
No, I registered via github and did nothing else :/
